ow_plexil/src/plexil-adapter/OwlatInterface.cpp

```diff
@@ -0,0 +1,111 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#include <thread>
+#include <vector>
+#include "OwlatInterface.h"
+
+using namespace owlat_sim_msgs;
+using std::hash;
+using std::string;
+using std::thread;
+
+using std::string;
+
+const string Name_OwlatUnstow = "/owlat_sim/ARM_UNSTOW";
+const string Name_OwlatStow =   "/owlat_sim/ARM_STOW";
+
+// Used as indices into the subsequent vector.
+enum LanderOps {
+  OwlatUnstow,
+  OwlatStow
+};
+
+static std::vector<string> LanderOpNames =
+  { Name_OwlatUnstow,
+    Name_OwlatStow
+  };
```

Comment:
I haven't seen this style:
```suggestion
static std::vector<string> LanderOpNames = {
  Name_OwlatUnstow,
  Name_OwlatStow
 };
```